frontend/lib/identification/util/canonical_json.dart

```diff
@@ -24,6 +24,8 @@ extension CanonicalJson on GeneratedMessage {
         map[field.tagNumber.toString()] = value;
       } else if (value is int) {
         map[field.tagNumber.toString()] = value.toString();
+      } else if (value is bool) {
+        map[field.tagNumber.toString()] = value.toString();
```

Comment:
Why do we need this? I think we should revert it.

Reply:
i couldn't write and read from `SecureStorage` without this check, because i added a bool to the protobuf

Reply:
Probably it was because you added the fields to cardInfo instead of userCode.